website/content/en/preview/troubleshooting.md

```diff
@@ -86,3 +86,42 @@ Your security groups are not blocking you from reaching your webhook.
 
 This is especially relevant if you have used `terraform-eks-module` version `>=18` since that version changed its security
 approach, and now it's much more restrictive.
+
+## Daemonsets can result in deployment failures
+
+For Karpenter versions 0.53 and earlier, daemonsets were not properly considered when provisioning nodes.
+This sometimes caused nodes to be deployed that could not meet the needs of the requested daemonsets.
+The result could be log messages like the following:
+
+```text
+Excluding instance type r3.8xlarge because there are not enough resources for daemons {"commit": "7e79a67", "provisioner": "default"}
+```
+
+One workaround is to set your provisioner to only use larger instance types.
+For more information, see [Issue #1084](https://github.com/aws/karpenter/issues/1084).
+Examples of this behavior are included in [Issue #1180](https://github.com/aws/karpenter/issues/1180).
+This issue was addressed in later Karpenter releases by [PR #1155](https://github.com/aws/karpenter/pull/1155).
+
+## Unspecified resource requests cause scheduling/bin-pack failures
+
+Not setting Kubernetes [LimitRanges](https://kubernetes.io/docs/concepts/policy/limit-range/) on pods can cause Karpenter to fail to schedule or properly bin-pack pods.
```

Comment:
```suggestion
Not using the Kubernetes [LimitRanges](https://kubernetes.io/docs/concepts/policy/limit-range/) feature to enforce minimum resource request sizes will allow pods with very low or non-existent resource requests to be scheduled.  This can cause issues as Karpenter bin-packs pods based on the resource requests.  If the resource requests do not reflect the actual resource usage of the pod, Karpenter will place too many of these pods onto the same node resulting in the pods getting CPU throttled or terminated due to the OOM killer.  This behavior is not unique to Karpenter and can also occur with the standard `kube-scheduler` with pods that don't have accurate resource requests.
```

Reply:
Do we have any best practice docs that recommend sizing of limits vs requests?

Reply:
@tzneal We have a general discussion on limits and pointers to Kubernetes docs on the subject in the [Karpenter Best Practices](https://aws.github.io/aws-eks-best-practices/karpenter/#use-limitranges-to-configure-defaults-for-resource-requests-and-limits) guide. Do you have suggestions for how to set particular limits?

Reply:
I incorporated your other comments.

Reply:
I don't know a good limit to request ratio value to recommend, probably worth punting on for now in that case as I can't find any recommendations from K8s or any existing EKS best practices either.

Reply:
Looks like most instance types are [here](https://aws.amazon.com/ec2/instance-types/#General_Purpose) have a max  cpu:memory ratio of 1:4. GPUs have 1:8. If we need a ratio, 1:4 would be a good place to start thinking about it.